pkg/scheduler/test_scheduler.go

```diff
@@ -840,18 +841,20 @@ func (s *Scheduler) triggerLogsStartEvent(ctx context.Context, id string) error
 
 func (s *Scheduler) triggerLogsStopEvent(ctx context.Context, id string) error {
 	if s.featureFlags.LogsV2 {
-		r, err := s.logsStream.Stop(ctx, id)
-		if err != nil {
-			s.logger.Errorw("can't send stop event for logs", "id", id, "error", err)
-			return err
-		}
+		go func() {
```

Comment:
sounds like the whole method is async, you can simply call go s.triggerLogsStopEvent(...)

Reply:
No it's sync - I want to know here that it was handled synchronously (to have better logging in case of issues)

Reply:
So we'll know here that event was handled by the logs-server end